packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -61,6 +61,12 @@ const ALLOWED_NEGATIVE_GOSSIPSUB_FACTOR = 0.1;
 // The Node should compute a recomended value every interval and log a warning
 // to terminal if it deviates significantly from the user's settings
 
+export type LodestarDiscv5Opts = Omit<IDiscv5CreateOptions, "enr" | "multiaddr"> & {
```

Comment:
We should not extend `IDiscv5CreateOptions` with Omit, Either select the few properties we want to expose or just use a new object. From looking into it we just need

```ts
export type Discv5Opts = {
  enr: string;
  bindAddr: string;
  bootEnrs: string[];
};
```